src/lib/plugins/players.js

```diff
@@ -17,45 +17,46 @@ module.exports.server = function (serv, { version }) {
     base: 'gamemode',
     aliases: ['gm'],
     info: 'to change game mode',
-    usage: '/gamemode <0-3> [player]',
+    usage: '/gamemode <mode> [player]',
```

Comment:
seems you now removed support completely for 0-3.
Vanilla handles both 0-3 and survival right?
maybe we could do the same?

Reply:
Since 1.13, vanilla stopped handling numeric game mode IDs.

Reply:
what about supporting them both?